kotlin/src/test/kotlin/tyler/StringCounterTest.kt

```diff
@@ -0,0 +1,33 @@
+package tyler
+
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class StringCounterTest {
```

Comment:
These are unit tests, this is what LeetCode does when you click the Submit button. Here, I write my own. These are very common for programmers to write.